samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -365,6 +366,15 @@ def template(self) -> Dict:
         """
         return self._template_dict
```

Comment:
do we want to get rid of this property and replace with the stack everywhere?

Reply:
Good catch! just verified it is no longer used

Reply:
Also remove the same thing from `BuildContext`, thx!